src/main/java/org/broadinstitute/hellbender/tools/exome/orientationbiasvariantfilter/OrientationBiasFilterer.java

```diff
@@ -362,4 +384,23 @@ public static VCFHeader createVCFHeader(final VCFHeader inputVCFHeader, final St
         final Set<String> sampleNameSet = samples.asSetOfSamples();
         return new VCFHeader(headerLines, sampleNameSet);
     }
+
+    static class UniqueIDWrapper<A>{
```

Comment:
If it would make the code cleaner you could drop the type parameter and just hardcode it as a genotype.  Probably want to add a comment explaining why we have this class.